### PR TITLE
feat: Social Icons element multiple improvements.

### PR DESCRIPTION
**Pull request by RajkiranBagal**
### Description
 - Trello task - https://trello.com/c/hPl8NAcl/278-social-icons
 
### Types of changes
New feature (non-breaking change which adds functionality)

### How has this been tested?
 - Multiple new improvements to test as per task provided.
 
### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/assets/js/ast-render-svg.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+		var viewBox = viewbox_array.join( " " )
+
+		return (
+			<svg xmlns="http://www.w3.org/2000/svg" viewBox={viewBox}><path d={path}></path></svg>
+		)
+	}
+
+}
+
+export default renderSVG
```
Comment (rohitp-bsf) [inline inc/assets/js/ast-render-svg.js:29]: Can we memorize this with React.memo() as we are just passing a param and fetching it?

File: inc/customizer/extend-custom-controls/src/social-icons/item-component.js [hunk 2]
```diff
@@ -1,14 +1,20 @@
 import {useState} from 'react';
 
 const {__} = wp.i18n;
-const {Dashicon, Tooltip, TextControl, Button} = wp.components;
+const {Dashicon, Tooltip, TextControl, Button } = wp.components;
+import FontIconPicker from "@fonticonpicker/react-fonticonpicker"
+import astIcons from "../../../../../assets/svg/ast-social-icons"
```
Comment (rohitp-bsf) [inline inc/customizer/extend-custom-controls/src/social-icons/item-component.js:6]: Is it possible to use an absolute path?

File: inc/customizer/extend-custom-controls/src/social-icons/social-component.js [hunk 3]
```diff
@@ -184,6 +189,8 @@ const SocialComponent = props => {
 		if (itemControl) {
 			let updateState = state.value;
 			let update = updateState.items;
+			let icon = ( 'email_2' === itemControl ) ? 'email' : ( 'phone_2' === itemControl ) ? 'phone' : itemControl;
```
Comment (rohitp-bsf) [inline inc/customizer/extend-custom-controls/src/social-icons/social-component.js:192]: can we just replace with regex  _ with digit with empty? `_/[0-9]+/g`

**Review by rohitp-bsf — CHANGES_REQUESTED** [review]

File: inc/customizer/extend-custom-controls/src/social-icons/social-component.js (hunk 3 above)
Comment (RajkiranBagal) [reply inc/customizer/extend-custom-controls/src/social-icons/social-component.js]: Done

File: inc/assets/js/ast-render-svg.js (hunk 1 above)
Comment (RajkiranBagal) [reply inc/assets/js/ast-render-svg.js]: Done

**Review by rohitp-bsf — APPROVED** [review]

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**